examples_tests/42.FragmentShaderPathTracer/main.cpp

```diff
@@ -8,6 +8,81 @@
 #include "../common/CommonAPI.h"
 #include "../common/Camera.hpp"
 #include "nbl/ext/ScreenShot/ScreenShot.h"
+#include "nbl/video/IPresentationOracle.h"
+
+class PresentationOracle : public video::IPresentationOracle 
```

Comment:
call it `Dumb` or `Simple` or `Naive` prefix with `C` and put it in `nbl/video/utilities`

then replace all similar code in all examples

Reply:
resolved